rumqttc/examples/async_manual_acks.rs

```diff
@@ -66,6 +62,11 @@ async fn main() -> Result<(), Box<dyn Error>> {
                 return Ok(());
             }
         }
+        if let Ok(Event::Outgoing(Outgoing::PubAck(puback))) = event {
+            if puback == 10 {
+                return Ok(());
+            }
+        }
```

Comment:
> Allow program to terminate after all pubacks have been sent
>
we don't need to terminate the program, we can just let it run normally. It will keep sending ping req and getting ping resp like other examples.

Reply:
Sounds good. Will remove this section.